### PR TITLE
Blog post - Thoughts about Training

### PR DESCRIPTION
**Pull request by liatsade**

### CODE REVIEW

**Comment by liatsade** [comment]
@RachelBaram please review.

**Comment by amitaibu** [comment]
@liatsade Interesting perspective. Few comments (apart of the required English corrections):

* I would drop the scrum part - mainly because it's not related 😉 . Scrum is a way for managing a project with a team. In my experience it's many times just the corporate way of doing agile. Those 10 minute talk, become an hour of boring conversations.
* Pushing you to give a session after ~2 weeks _is_ The Gizra Way™️ . Feeling uncomfortable, and being forced (first by us, later by your own inner voice), to try to do the best in a given time/ effort, IMO is similar to throwing a developer to a full blown project after one week of training. It's the see one, do one, teach one - on steroids
* I like the part you say that after 3 month you already did 12 classes on 4 different topics. I think this isn't trivial. By constantly doing an iterating over the process, you are able to improve much quicker then if you would try to polish the same class for 3 month.

**Comment by liatsade** [comment]
Thanks for the comments!
I do not agree Scram is not related. In the narrow sense - you're right, this is indeed a way to manage a project with a team, but to me (I have no practical experience with Scrum, but from what I read, and I have read a lot) it's much more than that. This is a way of thinking. It based on Agile principles and I found a lot of things in that way of thinking that are similar to The Gizra Way. 
It opened my mind and made me think differently about the whole thing of Employee Development. Perhaps you are not aware of it, but, for example, to push me to do training session right from the start, and improve it while doing it, it is completely a Scrum method :-)

**Comment by amitaibu** [comment]
> it is completely a Scrum method

Be that as it may, I find that in general, people think scrum = 10 minutes standup in the morning.

So if you feel scrum is relevant, I think it's worth adding a note or two that you refer to the actual scrum methodology, not just the practice.

I have seen more than once, big corporates applying scrum with scrum master. I assure you there was nothing agile or radical about it.

**Comment by RachelBaram** [comment]
@liatsade Are you planning on making any changes? I prefer editing after that. ;)

**Comment by liatsade** [comment]
Yes, I realise that the connection between Scrum and training is mostly in my head, and I need to explain it more clearly in the post. I'm still thinking how :-)
I'll update you when it's ready.

**Comment by amitaibu** [comment]
Would be great if you could finish this today - I like publishing posts on Thursday, and I think yours is almost ready.

**Comment by liatsade** [comment]
Ok, great. I'll do it.

**Comment by liatsade** [comment]
@amitaibu I added some explanation about Scrum, I hope it clarify the way I see it relevant to the training.
@RachelBaram ready for review.

File: app/_posts/2016-10-07-thoughts-about-training.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+title: "What happened when I thought they are playing Rugby in the meeting room? - Thoughts on how to set up a training program for a small company.
+"
+tags:
+  - The Gizra Way
+  - Training
+permalink: "/content/thoughts-about-training"
+layout: post
+author: LiatSade
+image:  ""
+description: "My job is to train and develop employees so that they can improve their skills and knowledge. This role exists in many big companies, but what is the best way to do it in a small and cool company like Gizra? I share with you my thoughts."
```
Comment (RachelBaram) [inline app/_posts/2016-10-07-thoughts-about-training.md]: @liatsade This should be a maximum of 160 characters. ;)

File: app/_posts/2016-10-07-thoughts-about-training.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+title: "What happened when I thought they are playing Rugby in the meeting room? - Thoughts on how to set up a training program for a small company.
```
Comment (RachelBaram) [inline app/_posts/2016-10-07-thoughts-about-training.md]: Please make this shorter (around max of 75 characters). See the blog roll page and how much space we have for a title.  People also don't like reading long titles. I would make sure that "training program" is included in the shortened version.

**Comment by RachelBaram** [comment]
@amitaibu I will only be able to get back to editing this tonight. Hope this does not spoil your publication plans.

**Comment by amitaibu** [comment]
No, its fine. It's not a hard deadline, just a good habit ;)

**Comment by RachelBaram** [comment]
@amitaibu Edits done and ready for your final review.

And as for good habits, I think we should try to have blog writers be ready on Wednesday end of the day, so I have Thursday morning to edit. On my end, I will set aside 3 hours every Thursday to support blog review.

**Comment by liatsade** [comment]
Thanks @RachelBaram for your good editing! Now I feel that post reflects much more accurately what I wanted to say.

File: app/_posts/2016-10-07-thoughts-about-training.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+---
+title: "Are they playing Rugby in the meeting room? Thoughts on setting up a training program."
+
+tags:
+  - The Gizra Way
+  - Training
+permalink: "/content/thoughts-about-training"
+layout: post
+author: LiatSade
+image:  ""
```
Comment (amitaibu) [inline app/_posts/2016-10-07-thoughts-about-training.md]: We need a thumbnail, and yo make sure the post looks ok when you grunt serve

**Comment by amitaibu** [comment]
Great post, really touching a lot of points that I struggle with as well. Just need a thumb image and I'll deploy.

**Comment by amitaibu** [comment]
(And a *great* English editing )

**Comment by amitaibu** [comment]
gizra_s_blog

* Apart of the thumb, the [author info](https://github.com/Gizra/gizra.com/blob/master/_config.yml#L37) and avatar are missing 
* Lets we have an image of the rugby without the `???` blue signs

File: app/_posts/2016-10-21-thoughts-about-training.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+---
+title: "Are they playing Rugby in the meeting room? Thoughts on setting up a training program."
```
Comment (amitaibu) [inline app/_posts/2016-10-21-thoughts-about-training.md]: Title is quite long. Also shouldn't add with a dot I think.

File: app/_posts/2016-10-21-thoughts-about-training.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+But how do we get there? This is a question that I still don't have a full answer for.
+So in the meantime, I will just share my thoughts.
+
+
+##Teach them to swim or throw them in the deep end?
+Do employees require organized training or should we just show them the pool and let them reveal things for themselves?
+In other words, who is responsible for learning, the employee or the trainer?
+
+When I came to Gizra, they threw me into the deep end. The water in Gizra is very deep! It was a frustrating experience. I know how to swim and I did all the moves, and yet I felt a lot of times that I was drowning. It was a new pool, a different water temperature, and it took time before I learned to take a rest in the shallow water, and where the ladder was located so I could get out. To tell you the truth, I'm still learning.
+For a new employee, it is a very inconvenient experience, even stressful, and I admit that sometimes I flirted with the thought of going back to my previous job, familiar and cozy, where I knew how to swim well.
```
Comment (amitaibu) [inline app/_posts/2016-10-21-thoughts-about-training.md]: You'll have to check how this is actually looking on the page (`grunt serve`), as you probably don't suspect it looks like this:

are_they_playing_rugby_in_the_meeting_room__thoughts_on_setting_up_a_training_program_

For a new paragraph you need a line break. For just a line break you need two spaces in the end of the sentence

```
something.  
like this
```

File: app/_posts/2016-10-21-thoughts-about-training.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+Was my first training session any good? No, and of course it could have been much better if I had more time to study the subject and prepare. But going back to the Scrum model, it was the best training version I could release at that time. I knew I had enough training sessions along the way, so I could improve while doing.
+If I did not jump into the deep end so early, I would probably, only now, be giving the first training session. I may have been more confident standing in front of the class, but moving forward at a much slower pace. Even this post couldn’t be written because I wouldn't have understood a lot of things I understand now.
+
+It is still hard to shake off old habits and even when I am writing these sentences I find it difficult to release and say, this is a good enough version of the post. There are a lot of open questions, and I still can not fully answer the question: how to set up a training program in a small company.
+But I remind myself to think "scrum" and say: "Let Go! Even in Rugby they don't stay with their arms interlocked all the time."
+
+
+
```
Comment (amitaibu) [inline app/_posts/2016-10-21-thoughts-about-training.md]: images are huge (We don't need 1.5MB for the rugby image). They should be resized to max 800px width

**Comment by amitaibu** [comment]
I've done a bit more edits, added the missing bio - and merged. Thanks!

**Comment by amitaibu** [comment]
Deployed -- http://www.gizra.com/content/thoughts-about-training/ 👍

**Pull request merged**